src/ckeditor.js

```diff
@@ -12,9 +14,56 @@
  * @singleton
  */
 
-CKEDITOR.define( function() {
+CKEDITOR.define( [ 'editor', 'mvc/collection', 'promise' ], function( Editor, Collection, Promise ) {
 	var CKEDITOR = {
 		/**
+		 * A collection containing all editor instances created.
+		 */
+		instances: new Collection(),
+
+		/**
+		 * Creates an editor instance for the provided DOM element.
+		 *
+		 * The creation of editor instances is an asynchronous operation, therefore a promise is returned by this
+		 * method.
+		 *
+		 *     CKEDITOR.create( '#content' );
+		 *
+		 *     CKEDITOR.create( '#content' ).then( function( editor ) {
+		 *         // Manipulate "editor" here.
+		 *     } );
+		 *
+		 * @param {String|Object} element An element selector or a DOM element, which will be the source for the
+		 * created instance.
+		 * @returns {Promise} A promise, which will be fulfilled with the created editor.
+		 */
+		create: function( element ) {
+			var that = this;
+
+			return new Promise( function( resolve, reject ) {
+				// If a query selector has been passed, transform it into a real element.
+				if ( typeof element == 'string' ) {
+					element = document.querySelector( element );
```

Comment:
> I agree with @Reinmar... still create() should accept a selector, because this is one of the most basic features. 

I really don't feel it's necessary. As a general approach I would choose to make less in our code (to make it smaller and simpler) and let the end-devs do the rest. The DOM isn't anymore a dark place where you need a tank to visit your grandma.

Other than that - I don't like one more thing - we spend time on implementing a solution for "create editor on all elements which match XYZ", when the more general problems is "create editor on all chosen elements". How do I create editors on all `.editor` elements inside a specific `.container` (to which I have a reference) if the `.container` class might be used in many places.

```
CKEDITOR.createAll( '.container .editor' );
```

Isn't what I want. Because what I want is:

```
$( mySpecificContainer ).find( '.editor' ).each( CKEDITOR.create );
```

And why did I use jQuery and not `querySelectorAll`? Ha... because `querySelectorAll` sucks, because it does not match the selector starting from the context, but with the entire tree (starting from document element). There are reasons why it was implemented this way, but basically it's not what you want. Therefore libs like jQuery create temporary id on the context element to be able to build a right selector.

Anyway, instead of trying to solve 90% of cases and making CKEditor bigger and bigger, let's try to solve 100% and make it as small as possible. This is my general approach to CKE5, not only applicable here. In this specific case the `createAll()` method won't harm anyone, so you can add, but as I said "I don't feel it's necessary".